### ISSUE TITLE
No method for One Call API

### ISSUE THREAD

**Issue by polar-webspark** [cmfcmf/OpenWeatherMap-PHP-API#174]
https://api.openweathermap.org/data/2.5/onecall?lat={lat}&lon={lon}&exclude={part}&appid={API key}
I need to call this api endpoint but there is no method for it

**Comment by mbomb007** [comment]
Related to #72's item "Support more / all OWM apis"

**Comment by mbomb007** [comment]
I agree, and this should be a high priority, as this only consumes a single API call. Since you only get 1,000 free API calls per day, getting everything in a single call is a huge gain.

**Comment by mbomb007** [comment]
I also noticed that this library is using the UV Index API, which was retired on 1st April 2021. https://openweathermap.org/api/uvi

They say the replacement is the One Call API.

**Comment by shmaltz** [comment]
+1 for this.

I need to get the 8 day forecast and with the OneCall API it is free.

This would be great!
